flink-table-store-core/src/main/java/org/apache/flink/table/store/file/format/FileFormat.java

```diff
@@ -97,10 +91,6 @@ public BulkFormat<RowData, FileSourceSplit> createReaderFactory(
         return createReaderFactory(rowType, projection, new ArrayList<>());
     }
 
-    public Optional<FileStatsExtractor> createStatsExtractor(RowType type) {
```

Comment:
Don't need to expose the `FileStatsExtractor` to the upper layer,  so we can just remove this method.